lib/diagnostic.rb

```diff
@@ -78,12 +86,12 @@ def initialize
 
 ## What will be the output from calling `HouseCat.new.say_hello`?
 # replace nil with your response
-Response.housecat_noise = nil
+Response.housecat_noise = "I am a Animal and I go nil"
 
 ## Explain why this would be the output, based on the method lookup chain.
 # your response as a comment here
 ##
-
+#House inherits from cat, which inherits from animal, so the inheritance comes directly from the superclass, which is not exactly what this reponse wants to be doing I think. A mxin would have been better.
```

Comment:
The inheritance chain is correct, but you didn't explain why the output is what it is.  `say_hello` doesn't exist on `HouseCat` or `Cat`, but does on `Animal`.  Since `say_hello` was found through the method lookup chain, it executes in the context of `HouseCat`, which is why `self.class.name` evaluates to `"HouseCat"`.